updateOrDeploy.sh

```diff
@@ -54,18 +55,25 @@ do
     typeset only_name=${dirName##*/}
     if [[ -e "${only_dir}" ]] 
     then
-        for cfgDir in $(find ${only_dir}  -maxdepth 1 -type d -name "${only_name}[0-9]*")
+        for baseDir in $(find ${only_dir}  -maxdepth 1 -type d -name "${only_name}[0-9]*")
         do
-            configDir="${cfgDir}/config/colors"
-            dir="${cfgDir}/colors"
-            if [ -e "$configDir" ]
+	    configDir="${baseDir}/config"
```

Comment:
Oops, looks like it. I edited in vi, probably there were some tabs in the original file.